pandas/tests/frame/test_stack_unstack.py

```diff
@@ -28,6 +28,9 @@ def future_stack(request):
 
 
 class TestDataFrameReshape:
+    @pytest.mark.filterwarnings(
+        "ignore:The previous implementation of stack is deprecated"
```

Comment:
Will there still be appetite to fix the `future_stack=False` case in 3.x? If not I would okay only testing `future_stack=True`

Reply:
I stated in https://github.com/pandas-dev/pandas/issues/57152#issuecomment-1918049286 that PRs to fix an issue with stack are welcome, but this made me rethink that. One potential consideration is that a bugfix can inadvertently introduce new issues. For this reason, I'm now thinking for a deprecated behavior we should prefer stability over bugfixes.

I'll remove the tests.

Reply:
Sorry - I've come full circle on this. If you're okay with it, I'd rather leave the tests in for now in case something unanticipated occurs.